realm/src/androidTest/java/io/realm/RealmTest.java

```diff
@@ -73,13 +76,20 @@ protected void setColumnData() {
     protected void setUp() throws Exception {
         Realm.deleteRealmFile(getContext());
         testRealm = Realm.getInstance(getContext());
-   }
+    }
 
     @Override
     protected void tearDown() throws Exception {
         if (testRealm != null) {
             testRealm.close();
         }
+
+        // Delete the encrypted Realm if present
+        File encryptedRealm = new File(getContext().getFilesDir(), ENCRYPTED_REALM_FILE_NAME);
```

Comment:
Only one unit test uses this. Better move it there. Just check at the start of the unit test and delete if needed?